stackprinter/formatting.py

```diff
@@ -152,30 +154,37 @@ def format_exc_info(etype, evalue, tb, style='plaintext', add_summary='auto',
                 msg += clr % chain_hint
 
         # Now, actually do some formatting:
-        msgs = []
+        parts = []
         if tb:
             frameinfos = [ex.get_info(tb_) for tb_ in _walk_traceback(tb)]
-            stack_msg = format_stack(frameinfos, style=style, reverse=reverse, **kwargs)
-            msgs.append(stack_msg)
+            if suppressed_exceptions and etype in suppressed_exceptions:
```

Comment:
I think it'd be better to convert `suppressed_exceptions` to a tuple and then do `issubclass(etype, suppressed_exceptions)`.

Reply:
great idea, will do that